rm/rm-server/src/test/resources/functionaltests/config/rm-with-db.ini

```diff
@@ -19,4 +19,5 @@ pa.rm.select.node.dynamicity=10000
 pa.rm.node.source.ping.frequency=10000
 pa.rm.client.ping.frequency=10000
 pa.rm.select.script.timeout=10000
-pa.rm.topology.enabled=false
```

Comment:
Why is it now required to enable topology in all configuration files even when it was not before? Is it because of the change in test `LocalSelectionTest`?